common/ta_errors.h

```diff
@@ -124,6 +124,10 @@ typedef enum {
   /**< Failed in cache operations */
   SC_CACHE_OFF = 0x03 | SC_MODULE_CACHE | SC_SEVERITY_MINOR,
   /**< Cache server is not turned on */
+  SC_CACHE_LOCK_ERROR = 0x04 | SC_MODULE_CACHE | SC_SEVERITY_FATAL,
```

Comment:
It is a bit difficult to distinguish from `SC_CACHE_LOCK_ERROR` and `SC_CACHE_LOCK`. Can you clarify or consolidate them?